Address final review comments (#27)
OpenLiberty/guide-testcontainers: README.adoc

You'll learn how to write true-to-production integration tests for Java microservices by using https://www.testcontainers.org/[Testcontainers^] and JUnit. You'll learn to set up and configure multiple containers, including the Open Liberty Docker container, to simulate a production-like environment for your tests.

> Sometimes tests might pass in development and testing environments, but fail in production because of the differences in how the application operates across these environments. Fortunately, these differences can be minimized by testing your application in the same Docker containers you will use in production. This approach helps to make the development, testing, and production environments more similar, enhancing test reliability.

Reviewer: ```suggestion
Sometimes tests might pass in development and testing environments, but fail in production because of the differences in how the application operates across these environments. Fortunately, these differences can be minimized by testing your application with the same Docker containers you will use in production. This approach helps to ensure parity across the development, testing, and production environments, enhancing quality and test reliability.
```

> Sometimes tests might pass in development and testing environments, but fail in production because of the differences in how the application operates across these environments. Fortunately, these differences can be minimized by testing your application with the same Docker containers you will use in production. This approach helps to ensure parity across the development, testing, and production environments, enhancing quality and test reliability.

Reviewer 2: Fortunately, you can minimize these differences by testing your application with the same Docker containers you use in production

=== What is Testcontainers?

> Testcontainers is an open-source library that provides containers as a resource at test time, creating consistent and portable testing environments. This is especially useful for applications that have external resource dependencies such as databases, message queues, or web services. By encapsulating these dependencies in containers, Testcontainers simplifies the configuration process and ensures a uniform testing setup that closely mirrors production environments.

Reviewer 2: open-source ---> open source

Also, add and configure the [hotspot=failsafe file=0]`maven-failsafe-plugin` plugin, so that the integration test can be run by the `mvn verify` command.

> As you started Open Liberty in dev mode, all the changes were automatically picked up. You can run the tests by pressing the `enter/return` key from the command-line session where you started dev mode. You see the following output:

Reviewer 2: As you started Open Liberty in dev mode
--->
When you started Open Liberty in dev mode
